src/events/EventsTooltip.js

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import { observer } from "mobx-react";
+import { Card } from 'antd';
+import { Col, Button } from 'antd';
+import eventConfig from './EventsConfig';
+import { ReactComponent as ClockIcon} from '../resources/svg/schedule-24px.svg';
+import { ReactComponent as TimerIcon} from '../resources/svg/timer-24px.svg';
+import { ReactComponent as SpeedIcon} from '../resources/svg/speed-24px.svg';
+import { ReactComponent as NoIcon } from '../resources/svg/not_interested-24px.svg';
+
+const TAG_TO_SVG = {
+  'speed': SpeedIcon,
+  'timer': TimerIcon,
+}
+
+const EventsTooltip = observer(({store}) => {
+
+  if(!store.tooltipResults && !store.mouseResults){
+    return null;
+  }
+
+  let graphics, title, extra;
+  if(store.mouseResults){
+    graphics = store.mouseResults.graphics;
+    title = store.mouseResults.title;
+    extra= <Button type="link" onClick={store.clearMouseResults}>X</Button>
+  } else {
+    graphics = [store.tooltipResults.graphic]
+    title = 'Events data:'
+  }
+  title = <span style={{fontSize: '16px'}}>{title}</span>
+
+  const style = {
+    display: 'block',
+    position: 'absolute',
+    top: '15px',
+    right: '55px',
+    width: '290px',
+    height: 'auto',
+    overflow: 'auto',
+    zIndex: '99',
+    // pointerEvents: 'none'
+  }
+  if (!graphics || graphics.length < 1) style.display = 'none';
+  
+  const results = {};
+  for (const prefix of Object.keys(eventConfig.statisticsFieldsInfo)) {
+    results[prefix] = {count: 0, sum: 0}
+  }
+
+  for(const graphic of graphics){
+      const attrs = graphic.attributes;
+      for (const prefix of Object.keys(results)) {
+          const value = attrs[prefix]
+          if (value !== null && value > 0) {
+              results[prefix].sum += value;
+              results[prefix].count++;
+          }
+      }
+  }
```

Comment:
Not sure if these lines are needed. I've seen only one instance of event per location, so maybe we can simplify this